cropduster/models.py

```diff
@@ -538,9 +538,11 @@ def _save_thumb(self, size, image=None, thumb=None, ref_thumb=None, tmp=False, c
 
         thumb_crop = thumb.crop(image, size)
         thumb_path = self.get_image_path(size.name, tmp=tmp)
-        thumb_image = thumb_crop.create_image(thumb_path, width=thumb.width, height=thumb.height)
 
-        if StandaloneImage:
+        if cropduster_settings.CROPDUSTER_CREATE_THUMBS:
```

Comment:
I think we'll need to exempt cropduster standalone images from this conditional. These are the images that can be uploaded via the ckeditor plugin. We render a single "thumb" for the image, and I think doing so is unavoidable. The alternative would be to somehow pass the thumbor url back to the ckeditor plugin, and that would require more changes in our image html processor that adds retina `srcset` attributes to images.

Reply:
Looks like the cropduster standalones are exempted by virtue of using a different method for their thumb creation- [_save_standalone_thumb](https://github.com/theatlantic/django-cropduster/blob/master/cropduster/models.py#L481).